components/suggestion/search_date_suggestion/index.js

```diff
@@ -8,12 +8,14 @@ import {getUserTimezone} from 'mattermost-redux/selectors/entities/timezone';
 
 import {getCurrentDateForTimezone} from 'utils/timezone';
 import {areTimezonesEnabledAndSupported} from 'selectors/general';
+import {getCurrentUserLocale} from 'selectors/i18n';
 
 import SearchDateSuggestion from './search_date_suggestion.jsx';
 
 function mapStateToProps(state) {
     const currentUserId = getCurrentUserId(state);
     const userTimezone = getUserTimezone(state, currentUserId);
+    const currentLocale = getCurrentUserLocale(state);
```

Comment:
I think this should be `getCurrentLocale`, not `getCurrentUserLocale`.

Reply:
Also, we could call the prop `locale` instead of `currentLocale` to save some typing

Reply:
Fixed!